prac_07/project.py

```diff
@@ -0,0 +1,25 @@
+COMPLETED_PERCENTAGE = 100
+
+
+class Project:
+    """Represent a Project object."""
+    def __init__(self, name="", start_date="", priority=0, cost_estimate=0, completion_percentage=0):
```

Comment:
The default value for cost_estimate could be a float (0.0). This won't impact the code functionality, however it shows the reader what type of value to expect for that attribute.